l1-contracts/contracts/dev-contracts/test/PriorityQueueTest.sol

```diff
@@ -5,38 +5,33 @@ pragma solidity 0.8.20;
 import "../../state-transition/libraries/PriorityQueue.sol";
 
 contract PriorityQueueTest {
-    // add this to be excluded from coverage report
-    function test() internal virtual {}
-
-    using PriorityQueue for PriorityQueue.Queue;
-
     PriorityQueue.Queue priorityQueue;
 
     function getFirstUnprocessedPriorityTx() external view returns (uint256) {
-        return priorityQueue.getFirstUnprocessedPriorityTx();
+        return PriorityQueue.getFirstUnprocessedPriorityTx(priorityQueue);
     }
 
     function getTotalPriorityTxs() external view returns (uint256) {
-        return priorityQueue.getTotalPriorityTxs();
+        return PriorityQueue.getTotalPriorityTxs(priorityQueue);
```

Comment:
Why?

Reply:
https://github.com/foundry-rs/foundry/pull/3128

`using for ` patterns are ignored by coverage

Reply:
You should add back, thanks!

Reply:
It moved to state-transition folder, so all good, missed it :)